### PR TITLE
Fix: doc issues and errors

### PR DESCRIPTION
**Pull request by rajasaad**
Fixed the documentation issues and errors from the release: 0.5.1.

### CODE REVIEW

File: tiatoolbox/dataloader/wsireader.py [hunk 1]
```diff
@@ -807,7 +807,7 @@ def read_rect(self, location, size, resolution=0, units="level"):
         return im_region
 
     def read_bounds(self, bounds, resolution=0, units="level"):
-        # Find parameters for optimal read
+        """ Find parameters for optimal read. """
```
Comment (shaneahmed) [inline tiatoolbox/dataloader/wsireader.py:810]: Remove space between """ and Find

File: tiatoolbox/dataloader/wsireader.py [hunk 2]
```diff
@@ -931,7 +931,7 @@ def __init__(self, input_img):
         self.glymur_wsi = glymur.Jp2k(filename=str(self.input_path))
 
     def read_rect(self, location, size, resolution=0, units="level"):
-        # Find parameters for optimal read
+        """  Find parameters for optimal read. """
```
Comment (shaneahmed) [inline tiatoolbox/dataloader/wsireader.py:934]: Remove space between """ and Find

File: tiatoolbox/dataloader/wsireader.py [hunk 3]
```diff
@@ -961,7 +961,7 @@ def read_rect(self, location, size, resolution=0, units="level"):
         return im_region
 
     def read_bounds(self, bounds, resolution=0, units="level"):
-        # Find parameters for optimal read
+        """ Find parameters for optimal read. """
```
Comment (shaneahmed) [inline tiatoolbox/dataloader/wsireader.py:964]: Remove space between """ and Find

File: tiatoolbox/dataloader/wsireader.py [hunk 4]
```diff
@@ -1097,7 +1097,7 @@ def _info(self):
         return param
 
     def read_rect(self, location, size, resolution=1.0, units="baseline"):
-        # Find parameters for optimal read
+        """ Find parameters for optimal read. """
```
Comment (shaneahmed) [inline tiatoolbox/dataloader/wsireader.py:1100]: Remove space between """ and Find

File: tiatoolbox/dataloader/wsireader.py [hunk 5]
```diff
@@ -1125,7 +1125,7 @@ def read_rect(self, location, size, resolution=1.0, units="baseline"):
         return im_region
 
     def read_bounds(self, bounds, resolution=1.0, units="baseline"):
-        # Find parameters for optimal read
+        """ Find parameters for optimal read. """
```
Comment (shaneahmed) [inline tiatoolbox/dataloader/wsireader.py:1128]: Remove space between """ and Find

**Review by shaneahmed — APPROVED** [review]

**Review by Srijay-lab — APPROVED** [review]

**Pull request merged**